docs/tech/rfc/0021-240711-evaluating-reliability.md

```diff
@@ -0,0 +1,161 @@
+software reliability and product stability. The immediate change would be to separate these
+sections out in linear rather than having a single Product and Engineering section.
+This helps us consolidate all outstanding issues in the existing product without the
+noise of future development plans. 
+
+It's important before pursuing new feature development, we evaluate our set of outastanding 
+maintenance and stability work. The idea is to handle the higher priority work before moving 
+on to work that expands functionality and thereby expands the space in which faults can arise 
+and outpace our stability efforts.
+
+## 2.1 - Qualification of bugs and stability improvements
+In order to better qualify the readiness of our software, it's important we have an indication 
+of the importance of any particular issue. I propose we use the following mapping 
+to the priority levels in linear and stay very disciplined in qualifying the urgency of tasks 
+in the following manner:
+
+Urgent (1): This bug or lack of feature will stop the user from making progress on their 
+hardware development and testing. Could cause significant lost of time and resources for 
+the customer. Examples are critical bugs which can cause complete failure of a system or
+ subsystem of the product.
+
+High (2): This bug or lack of feature significantly reduces quality of a user's experience 
+and are important to get done before any new feature development is completed. These are 
+bugs which significantly slow down work or could lead to repeated work.
+
+Medium(3): This bug or lack of feature makes a noticeable impact to the user's experience 
+and should be targeted to be addressed in the next release. Alternatively, these are also 
+bugs which have a decent workaround for them but must be addressed eventually.
+
+Low(4):  These bugs have low impact on the user and could be put off to future releases if necessary.
+     
+## 2.2 - Software Ownership
+It is important we create clear segments of responsibility across the codebase. Having a 
+single owner allows for a clear path for delegation. Further, it is that engineers responsibility
+to have a mechanism or process in place to evaluate stability and reliability of their software.
+
+In general, it is important that everyone is responsible for their own software. That is, 
+if they introduce a bug, or instability, it is their responsbility to address the problem. 
+This is ultimately is the most productive way to address issues as the RE will have the most 
+context for solving the problem. 
+
+# 2.3- Metrics from active product use
+One setback in operating in the aerospace industry and only hosting Synnax on prem is the 
+inability to receive logs from crashes,errors of our software. Not having a single automated
+source for this information will become increasingly problematic, particularly when customers
+begin to outnumber of available engineers that can be mobilized forreal time support. Further, 
```

Comment:
Add a space in "forreal" and change wording to "begin to outnumber available engineers".

Reply:
fr